src/PackageMatcher.ts

```diff
@@ -45,14 +45,41 @@ export default class PackageMatcher extends Array<Package> {
   }
 }
 
+export interface FunctionGroup {
+  names?: string[];
+  labels?: string[];
+}
+
 export interface Package {
   path?: string;
   exclude?: string[];
   module?: string;
   shallow?: boolean;
+  functions?: FunctionGroup[];
   prisma?: string; // custom prisma client module id
 }
 
+function parseFunctions(functions: unknown): FunctionGroup[] | undefined {
+  if (!functions || !Array.isArray(functions)) return;
+
+  const result: FunctionGroup[] = [];
+
+  for (const fun of functions as unknown[]) {
+    const entry: FunctionGroup = {};
+    if (typeof fun === "object" && fun !== null && ("name" in fun || "names" in fun)) {
+      if ("name" in fun) entry.names = [String(fun.name)];
+      if ("names" in fun) entry.names = Array.isArray(fun.names) ? fun.names : [String(fun.names)];
+      if ("label" in fun) entry.labels = [String(fun.label)];
+      if ("labels" in fun)
```

Comment:
It probably makes sense to be more lenient with this and interpret `name` and `names` the same (ditto for `label` and `labels`). If someone passes an array as `name` they meant `names`, not a single `foo,bar` name.